rust/pattern/constraint/concept.rs

```diff
@@ -41,18 +51,12 @@ impl From<String> for IsConstraint {
 
 impl From<UnboundVariable> for IsConstraint {
     fn from(var: UnboundVariable) -> Self {
-        Self::from(var.into_concept())
-    }
-}
-
-impl From<ConceptVariable> for IsConstraint {
-    fn from(var: ConceptVariable) -> Self {
-        Self { variable: Box::new(var) }
+        Self::new(var.into_concept())
     }
 }
```

Comment:
This could never be used in a builder, so it's now a private constructor instead (`new()`).